ui/pages/create-account/connect-hardware/index.js

```diff
@@ -72,44 +67,16 @@ class ConnectHardwareForm extends Component {
     device: null,
   };
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { accounts } = nextProps;
-    const newAccounts = this.state.accounts.map((a) => {
-      const normalizedAddress = a.address.toLowerCase();
-      const balanceValue = accounts[normalizedAddress]?.balance || null;
-      a.balance = balanceValue ? formatBalance(balanceValue, 6) : '...';
-      return a;
-    });
-    this.setState({ accounts: newAccounts });
-  }
```

Comment:
Unsafe method removed as all it was doing is creating a `computed` property. Method is easily replaced with the line in `account-list.js`:

```js
accounts: formatAccounts(props.accounts, getMetaMaskAccounts(state)),
```